control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -618,11 +617,11 @@ func (r *reconciler) reconcileConfig(ctx context.Context, hcp *hyperv1.HostedCon
 		errs = append(errs, fmt.Errorf("failed to reconcile proxy config: %w", err))
 	}
 
-	icsp := globalconfig.ImageContentSourcePolicy()
+	icsp := globalconfig.ImageDigestMirrorSet()
```

Comment:
This is where you'd need to delete any existing ImageContentSourcePolicy first, and then reconcile the ImageDigestMirrorSet